src/main/java/org/jabref/gui/util/WelcomePage.java

```diff
@@ -0,0 +1,75 @@
+package org.jabref.gui.util;
+
+import javafx.geometry.Pos;
+import javafx.scene.control.Hyperlink;
+import javafx.scene.control.Label;
+import javafx.scene.layout.VBox;
+import javafx.scene.text.Text;
+import javafx.scene.text.TextAlignment;
+import javafx.scene.text.TextFlow;
+
+import org.jabref.gui.ClipBoardManager;
+import org.jabref.gui.DialogService;
+import org.jabref.gui.StateManager;
+import org.jabref.gui.frame.JabRefFrame;
+import org.jabref.gui.importer.NewDatabaseAction;
+import org.jabref.gui.importer.actions.OpenDatabaseAction;
+import org.jabref.gui.preferences.GuiPreferences;
+import org.jabref.gui.undo.CountingUndoManager;
+import org.jabref.logic.ai.AiService;
+import org.jabref.logic.util.TaskExecutor;
+import org.jabref.model.entry.BibEntryTypesManager;
+import org.jabref.model.util.FileUpdateMonitor;
+
+/**
+ * A simple Welcome Page for the JabRef application.
+ */
+public class WelcomePage extends VBox {
+
+    public WelcomePage(JabRefFrame frame,
+                       GuiPreferences preferences,
+                       AiService aiService,
+                       DialogService dialogService,
+                       StateManager stateManager,
+                       FileUpdateMonitor fileUpdateMonitor,
+                       BibEntryTypesManager entryTypesManager,
+                       CountingUndoManager undoManager,
+                       ClipBoardManager clipBoardManager,
+                       TaskExecutor taskExecutor) {
+
+        setAlignment(Pos.CENTER);
+        setSpacing(10);
+
+        // Title
+        Label welcomeLabel = new Label("Welcome to JabRef!");
```

Comment:
This is not translated. You need to use `Localization.lang`. See https://devdocs.jabref.org/code-howtos/localization.html#localization for details.

@InAnYan How did you do it in the AiTab with links and sentences? Did we come up with a good solution.

@HaynesCrossley-ANU I am asking for help here, because for translation, we aim for translating whole sentences. Reason: In different languages, the ordering of words within a sentence is different. (Because the grammar differs from langauge to langauge)